c7n/resolver.py

```diff
@@ -134,7 +137,10 @@ def get_values(self):
         if format == 'json':
             data = json.loads(contents)
             if 'expr' in self.data:
-                return jmespath.search(self.data['expr'], data)
+                res = jmespath.search(self.data['expr'], data)
+                if res is None:
+                    log.warning('ValueFilter: %s key returned None' % self.data['expr'])
```

Comment:
s/ValueFilter/ValueFrom .. we reuse the resolver implementation across many filters/etc.